web/src/resize.js

```diff
@@ -145,10 +145,17 @@ export function resize_sidebars() {
     return h;
 }
 
+export function resize_table_fix_head_max_height() {
+    const recent_topics_filter_height = $("#recent_topics_filter_buttons").height();
+    // 93 = navbar height (40px) + padding (12px) + compose height (41px)
```

Comment:
Wouldn't it be better to just calculate these values using JS too?

Reply:
The navbar height reduces on the small screens but it does not have an effect on the total area available for the table. Using js to calculate the navbar height and subtracting it from total height (100vh) increases the available height for a table on small screens (when the height of the navbar is reduced) and then the behavior is not right.

Plus, padding is given to some parent of the table for which I do not think it would be right to add additional js to extract that padding value. Also, this is how it is done in the css too (hard coding values).

Taking all these into account I thought it would be a better idea to hard code them. Still, if you think the other approach is better, we can take a look at that angle too.

Reply:
We can change values in other places if that will make it more maintainable, since I am sure there will be changes to their height in the future, and we can simply forget to update it here.

Reply:
@amanagr I've addressed your feedback. Thanks for pointing that out!